bin/pca-import

```diff
@@ -170,27 +172,31 @@ def import_assessment(request, source, force=False):
     return True
 
 
-def import_json_file(filename, force, customerImport, assessmentImport):
+def import_json_file(filename, force, customer_import, assessment_import, user_report_import):
```

Comment:
Do you intend this function to perform only one type of import?  What if someone passes `customer_import = True` and `assessment_import = True`?  I see that the code will only perform the customer import, but this may not be what the user expects.

The best way to address this would be for the function to accept a single `enum` value (`enums` are backported to Python 2.7 via [this `pip` package](https://pypi.org/project/enum34/)) value instead of the three Boolean values.  That said, I'm willing to approve if you create a docstring for this function describing that it expects _exactly one_ of the three Booleans to be `True`.

Reply:
See a8014ba675ba14a104152964cb72cf7ce0bb7127 for my changes. I made it an enum value instead. It cleaned up the code somewhat